packages/react/src/lib/accessibility/Behaviors/Tree/treeTitleBehavior.ts

```diff
@@ -21,6 +23,12 @@ const treeTitleBehavior: Accessibility = (props: any) => ({
       performClick: {
         keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
       },
+      expandOrFocusFirstChild: {
+        keyCombinations: [{ keyCode: keyboardKey.ArrowRight }],
+      },
+      collapseOrFocusParent: {
+        keyCombinations: [{ keyCode: keyboardKey.ArrowLeft }],
+      },
```

Comment:
do you need to handle these in tree title? I think you should be able to handle these cases in 1 place -  in TreeItem

Reply:
On left/right when I expand/collapse, I am invoking the `onClick` handler. The only way to get the whole event chain (`Tree`, `TreeItem`, `TreeTitle`) is from `TreeTitle`, otherwise I may need to do another props-callback magic.

I would rather do the collapse/expand here and let `TreeItem` to handle the focus. What do you think?